python/graphscope/framework/graph.py

```diff
@@ -156,6 +164,14 @@ def _close_learning_instances(self):
             instance.close()
         self._learning_instance_list.clear()
 
+    def _launch_interactive_instance_impl(self):
+        try:
+            sess = get_session_by_id(self.session_id)
+            sess.gremlin(self)
```

Comment:
Maybe we should add a option to control whether automatically create interactive instance whenever a graph is loaded
Reason:  If always create, when user want to get the interactive instance, they call `sess.gremlin(graph, params)`, and the method `sess.gremlin()` will simply find in the cache and return, the **params** is silently ignored, which may be confusing.  Also since user can never reach the logic of creating instance, it should be in another private method.

Reply:
There may be another circumstance that user can close a previous instance, and create a new one. We allow that. And this procedure of course will takes about ~30s. To prevent user wondering why two gremlin call consumed very different time, I think you should add the silent thread creating technique to somewhere in the doc, and in the session/graph comments.

Reply:
Add `auto_create_interactive_engine`  in config.py. Defaults to True.

Reply:
>  To prevent user wondering why two gremlin call consumed very different time, I think you should add the silent thread creating technique to somewhere in the doc, and in the session/graph comments.

Apart from the doc, when user run `.gremlin(sess, params)` with params, (we know the params are ignored), maybe we should print a warning to remind user that the params is not taken effect. And let them know the backend behavior.

Reply:
Nice advice. I have updated doc of `gremlin`, which clarify how to create a new instance under the same graph with different engine_params. 
I didn't add `logger.warning` in code, as `gremlin` may run in a thread and hidden from users.

Reply:
A new issue opened to trace InteractiveQuery create failed after close.